libs/metrics/top_level_metrics.py

```diff
@@ -297,11 +296,11 @@ def calculate_covid_patient_ratio(data: pd.DataFrame, region: Region):
         cdc_covid_patient_ratio = data[CommonFields.BEDS_WITH_COVID_PATIENTS_RATIO_HSA]
         first_cdc_ratio_index = cdc_covid_patient_ratio.first_valid_index()
     else:
-        cdc_covid_patient_ratio = pd.Series(dtype=float)
+        cdc_covid_patient_ratio = EMPTY_TS
```

Comment:
Out of curiosity, does this actually behave differently or was it just cleaner?

Reply:
This behaves the same, I just didn't see that `EMPTY_TS` existed when first making these changes